src/app/base/components/node/HardwareCard/_index.scss

```diff
@@ -0,0 +1,26 @@
+@mixin HardwareCard {
+    .hardware-card {
+        padding: 0.5rem 1rem 1rem 1rem;
+        
+        .p-list__item + .p-list__item {
+            @include pseudo-border(top);
+        }
+
+        li.p-list__item {
+            padding: 0.5rem 0 !important;
+        }
+
+        li.p-list__item:first-child {
```

Comment:
I don't think you'd need this either if you use `p-list--divided`.

Reply:
Dead on. That's so much better